Elevato/Extand/ExtandEvents.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Elevator.Extand
+{
+    public static class ExtandEvents
+    {
+        public static object Raise(this MulticastDelegate multicastDelegate, object sender, object eventArgs)
+        {
+            object retval = null;
+            if (multicastDelegate != null)
```

Comment:
if (multicastDelegate == null) return null would be better here - less nesting

Reply:
not sure why i want that but it looks like there is more line in compare of last time